tests/test_socket_server.py

```diff
@@ -99,16 +107,15 @@ async def test_exit(self):
         await asyncio.sleep(0.5)
         self.assertFalse(self.srv.connected)
 
-    async def test_full_command_sequence(self):
-        name = "Test1"
-        channels = 1
+    async def socket_server_test(self):
```

Comment:
Another foo_test. Please document what this does.

Reply:
Done